go.mod

```diff
@@ -57,14 +62,15 @@ require (
 	github.com/bigkevmcd/go-configparser v0.0.0-20200217161103-d137835d2579
 	github.com/go-kit/kit v0.11.0
 	github.com/gobwas/glob v0.2.3
-	github.com/google/cadvisor v0.36.0
+	github.com/google/cadvisor v0.44.0
```

Comment:
#430 requests to bump up cadvisor to this version. It sounded like this upgrade fixes their issue, but I'm still a little iffy about jumping up to a pre-release version of cadvisor. What kind of testing was done for this?